utils/build-master.config

```diff
@@ -0,0 +1,9 @@
+[component]
+component=iof
+
+[commit_versions]
+CART = 0b6f9a14ca4bf6b045ecf078c64e812c669996f6
```

Comment:
This isn’t master

Reply:
See earlier comments, we need https://github.com/daos-stack/iof/pull/16 landed before we can actually switch to master.